src/_help/customization-options/operations-navigation.md

```diff
@@ -5,7 +5,9 @@ title: Operations and navigation
 - TOC
 {:toc}
 
-Bump.sh offers several methods for sorting your API operations to provide a smooth and coherent reading structure, based on either the tags or operation names and URLs.
+Bump.sh offers several methods for sorting your API operations to provide a smooth and coherent reading structure based on tags or operation names and URLs.
+
+This setting is available under the "Customize UI" settings tab of a doc or under the "Customize UI" settings tab of the doc's Hub.
```

Comment:
```suggestion
From the “Customize UI” tab in your documentation settings, you can select a custom operations & navigation grouping mode between either “Automatic”, “By Path” or “By Tag”.
```

Reply:
I suggest to use a similar wording as other customization doc pages (from https://docs.bump.sh/help/customization-options/color-logo-meta-images/ for example)